server/config/incident.js

```diff
@@ -10,6 +10,7 @@ const paths = {
   staffInvolved: bookingId => `/report/${bookingId}/staff-involved`,
   staffMemberName: bookingId => `/report/${bookingId}/staff-member-name`,
   staffNotFound: bookingId => `/report/${bookingId}/staff-member-not-found`,
+  selectStaffMember: bookingId => `/report/${bookingId}/select-staff-member`,
```

Comment:
I think these URLs might need updating. 
See [Andy Hudson's comment on DCS-403 at 03:26 on 09.09.2020](https://dsdmoj.atlassian.net/browse/DCS-403?focusedCommentId=114305&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-114305)

Reply:
as discussed, that comment no longer applies